core/controlplane/config/config.go

```diff
@@ -365,9 +365,15 @@ func (c *Cluster) ControllerFeatureGates() model.FeatureGates {
 	if !c.Experimental.Admission.Priority.Enabled {
 		gates["PodPriority"] = "false"
 	}
+	if c.Experimental.Admission.Priority.Enabled {
```

Comment:
I think these can all be changed to this form:
`gates["PodPriority"] = c.Experimental.Admission.Priority.Enabled`

Reply:
@kevtaylor Good spot, fixed now